<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node pkg="suction_pump" type="suction_pump_action_server" name="suction_pump_fxp" output="screen"/>
</launch>